backend/accounts/baseaccount.go

```diff
@@ -59,6 +59,8 @@ type AccountConfig struct {
 	GetSaveFilename       func(suggestedFilename string) string
 	// Opens a file in a default application. The filename is not checked.
 	UnsafeSystemOpen func(filename string) error
+	// BtcCurrencyUnit is the unit which should be used to format fiat amounts values expressed in BTC..
```

Comment:
Currently, it is `BtcUnit` for backend configuration, `formatUnit` for btc/coin and `BtcCurrencyUnit` for accounts/baseaccount. The rationale is the following:
- `formatUnit` express the formatting unit of a BTC coin (while `unit` is the main unit)
- `BtcCurrencyUnit` express the unit of BTC when it is intended as a currency value (i.e. in amount conversions)
- `BtcUnit`is the backend configuration which is used to store the sat configuration for btc coins and currency, since we decided that we want the same behavior for both of them.

If we decide on a different behavior in the future, having separated names for coins and currency formatting could be helpful, I think. But if you feel like it is better to have a consistent name everywhere I am ok with that.